src/Speckle.Objects/Other/Instance.cs

```diff
@@ -149,13 +149,14 @@ protected override IEnumerable<Base> GetTransformableGeometry()
   public Plane GetInsertionPlane()
   {
     // TODO: UPDATE!
```

Comment:
This vague TODO is a bit scary... Are we missing anything critical that is yet to be implemented here?

I'm aware this is unrelated to this PR. Just asking

Reply:
![image](https://github.com/user-attachments/assets/a37dabca-34ea-497e-bcec-6854dd2cf268)

These are some very old comments from @clairekuang . I suspect they are not a problem.
But @clairekuang if you don't mind having a quick look in-case there is indeed some gaps that need filling. (low priority)

Reply:
I'm ok removing the comments, if any bugs are reported with non-0,0,0 based base points, we can address them at that point